### PR TITLE
debug_branch

### PR DESCRIPTION
**Pull request by ekengni**

### CODE REVIEW

File: src/main/webapp/gallery.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
```
Comment (lazau-g) [inline src/main/webapp/gallery.html]: You don't need to include script.js in this page. It only affects pages with 'greeting-container' element.

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  </head>
+  <body>
+    <div class = "heading-container">
+        <div class = "heading-wrapper">
+            <div class = "left-heading">
+                <div class = "nav-link-wrapper">
+                    <a href = "about_me.html"> About Me </a>
+                </div>
+                <div class = "nav-link-wrapper">
+                    <a href = "projects_research"> Experience </a>
```
Comment (lazau-g) [inline src/main/webapp/index.html]: projects_research -> projects_research.html

**Review by lazau-g — COMMENTED** [review]
You'll need to add a target/ line to your .gitignore as the target directory contains build artifacts generated by maven that can be regenerated.

File: src/main/webapp/gallery.html (hunk 1 above)
Comment (ekengni) [reply src/main/webapp/gallery.html]: That is. My commit add my entire directory, so there is some work that has not been completed

**Review by lazau-g — APPROVED** [review]
Great job!

**Pull request merged**